src/components/shortcodes/simpleTab.js

```diff
@@ -0,0 +1,103 @@
+import React, { useEffect, useState } from 'react'
+import PropTypes from 'prop-types'
+import '../../styles/components/simpletab.scss'
+import { useLocation } from '@reach/router'
+
+const SimpleTab = React.memo(({ children }) => {
+  const location = useLocation()
+  const selectedTab = location.hash ? location.hash.split('#')[1] : null
+
+  const [value, setValue] = useState(0)
+  const [tabLabelList, setTabLabelList] = useState([])
+  const [tabPanelList, setTabPanelList] = useState([])
+  const multiTabs = children.length ? true : false
+
+  useEffect(() => {
+    const _tabLabelList = []
+    const _tabPanelList = []
+
+    if (multiTabs) {
+      children.forEach((child) => {
+        _tabLabelList.push(child.props.label)
+        _tabPanelList.push(child.props.children)
+      })
+    }
+
+    setTabLabelList(_tabLabelList)
+    setTabPanelList(_tabPanelList)
+
+    if (_tabLabelList.includes(selectedTab)) {
+      const selectedTabIdx = _tabLabelList.findIndex((el) => el === selectedTab)
+      setValue(selectedTabIdx)
+    } else {
+      setValue(0)
+    }
+  }, [children, multiTabs, selectedTab])
+
+  function a11yProps(type, index) {
+    if(type === 'panel') {
+      return {
+        id: (`${tabLabelList[index]}`).replace(/\s/g, '-'),
+        'aria-labelledby': (`${tabLabelList[index]}-tab`).replace(/\s/g, '-'),
+        role: "tabpanel"
+      }
+    } else {
+      return {
+        id: (`${tabLabelList[index]}-tab`).replace(/\s/g, '-'),
+        'aria-controls': (`${tabLabelList[index]}`).replace(/\s/g, '-'),
+        role: 'tab'
+      }
+    }
+  }
+
+  const TabPanel = React.memo((props) => {
```

Comment:
It's no need to wrap the panel component with the `memo` func because the props won't always keep the original value.